app/src/main/java/nl/tudelft/b_b_w/view/ContactAdapter.java

```diff
@@ -83,9 +83,9 @@ private int getImageNo(int trust) {
      */
     private int calculateImageIndex(int trust) {
         final int trustInterval = 20;
-        int result = trust/ trustInterval - 1;
+        double result = trust/ trustInterval - 0.5;
```

Comment:
I have made an issue for this, because this function does not yet calculate the values I desire.